packages/subgraph/tests/dao/dao_v1_3_0.test.ts

```diff
@@ -250,7 +248,15 @@ describe('handleExecuted', () => {
 
   describe('ERC20 action', () => {
     beforeAll(() => {
-      createTokenCalls(DAO_TOKEN_ADDRESS, 'name', 'symbol', '6', '10');
+      const totalSupply = '10';
+      const decimals = '6';
+      createERC20TokenCalls(
+        DAO_TOKEN_ADDRESS,
+        totalSupply,
+        'name',
+        'symbol',
+        decimals
+      );
```

Comment:
if the name, symbol, totalSupply & decimal are the same in this test ? why don't you make it global and reuse it everywhere?

Reply:
I agree